Makefile

```diff
@@ -408,6 +408,7 @@ bundle-push: ## Push the bundle image.
 
 .PHONY: release-build
 release-build:
+	yq '.[].version'  hack/release.yaml | grep 'latest' && exit 1
```

Comment:
grep return with `1` if no matches. No need for: ` && exit 1`

Reply:
But if we *find* 'latest' we *want* it to fail...

Reply:
right, you still need to add `|| true` at the end

`yq '.[].version' hack/release.yaml | grep -q 'latest' && exit 1 || true`

Reply:
Good catch! Fixed.